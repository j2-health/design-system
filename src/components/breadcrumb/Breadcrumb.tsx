import { Breadcrumb as AntdBreadcrumb, BreadcrumbProps } from 'antd'

export type Props = Expand<BreadcrumbProps>

const Breadcrumb = ({ items, ...props }: Props) => {
  return <AntdBreadcrumb {...props} items={items} />
}

export { Breadcrumb }
