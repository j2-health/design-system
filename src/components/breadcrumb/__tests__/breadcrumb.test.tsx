import { render } from '@testing-library/react'
import { Breadcrumb } from '../Breadcrumb'

describe('Breadcrumb', () => {
  it('should render correctly', () => {
    const { container } = render(<Breadcrumb items={[]} />)
    expect(container).toMatchSnapshot()
  })
})
