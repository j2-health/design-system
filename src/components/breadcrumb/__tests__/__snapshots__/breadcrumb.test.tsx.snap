// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb should render correctly 1`] = `
<div>
  <nav
    class="ant-breadcrumb css-dev-only-do-not-override-os2fvf"
  >
    <ol />
  </nav>
</div>
`;
